deployment/hasura/migrations/AerieMerlin/23_activity_directive_changelog/up.sql

```diff
@@ -0,0 +1,904 @@
+-- add last_modified_by to activity_directive
+alter table activity_directive
+  add column last_modified_by text,
+
```

Comment:
Minor: Extra newline

Reply:
This was an intentional delineation between columns and constraints, as is done throughout many Aerie existing `sql` files.